<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyWidget_DropCapBase</class>
 <widget class="QFrame" name="PropertyWidget_DropCapBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
   <item row="1" column="0">
    <widget class="QLabel" name="dropCapLinesLabel">
     <property name="toolTip">
      <string>Ensure that first lines of a paragraph won't end up separated from the rest (known as widow/orphan control)</string>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ScrSpinBox" name="dropCapLines">
     <property name="suffix">
      <string> lines</string>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="dropCapOffsetLabel">
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="dropCapBox">
     <property name="text">
      <string>use Drop Caps</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="dropCapCharStyleLabel">
     <property name="text">
      <string>Use Character Style</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="CharStyleComboBox" name="dropCapCharStyleCombo"/>
   </item>
   <item row="2" column="1">
    <widget class="ScrSpinBox" name="dropCapOffset">
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>CharStyleComboBox</class>
   <extends>QComboBox</extends>
   <header>ui/spalette.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
